### PR TITLE
Login

### PR DESCRIPTION
**Pull request by parinitha25**
Register and login page completed

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2393,6 +2393,11 @@
       "resolved": "https://registry.npmjs.org/boolbase/-/boolbase-1.0.0.tgz",
```
Comment (vsanjeevagowda-s) [inline package-lock.json]: Please add the package-lock.json to .gitignore and follow the same for all projects

File: src/Action/alert.action.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const failureAlert = (error) => {
+  return {
+    type: FAILURE_ALERT,
+    error,
+  }
+}
+
+const formateMessage = (data) => {
+  if(typeof(data) === 'string'){
+    console.log(data)
```
Comment (vsanjeevagowda-s) [inline src/Action/alert.action.js]: PLease remove console.log from the project

File: src/Action/api/api.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      return fetch(`${ROOT_API_URL}${path}`,{
+        headers: headers(),
+      })
+        .then(resp => resp.json())
+        .then(resp => resolve(resp))
+        .catch(error => reject(error))
+    })
+  },
+  post: (path, payload) => {
+      debugger
```
Comment (vsanjeevagowda-s) [inline src/Action/api/api.js]: Remove debugger from project

File: src/Components/homeComponent.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+import '../CSS/signupComponent.css';
+ 
+class homeComponent extends Component {
+    render() {
+        return (
+            <div><h1 className="home">Home</h1>
+                <img src={require('../images/website.jpg')} alt="image" className="homeimage" />
```
Comment (vsanjeevagowda-s) [inline src/Components/homeComponent.js]: Keep images in public folder
SOLUTION Refrence: https://stackoverflow.com/questions/47196800/reactjs-and-images-in-public-folder/48730272

**Review by vsanjeevagowda-s — CHANGES_REQUESTED** [review]
PLease have a look at the comments added

File: package-lock.json (hunk 1 above)
Comment (parinitha25) [reply package-lock.json]: ok

File: src/App.js [hunk 5]
```diff
@@ -1,18 +1,54 @@
...
+       </Switch>
+       </Router>
+
+      </Fragment>
+    );
+  }
+};
+
+const mapStateToProps = state => {
+  console.log('this: ', this);
```
Comment (vsanjeevagowda-s) [inline src/App.js]: Please remove console.log

**Pull request merged**